code/modules/projectiles/ammo_datums.dm

```diff
@@ -241,53 +241,54 @@
+		SPAN_HIGHDANGER("[user] aims \the [fired_from] directly at your head!"),
+		SPAN_DANGER("[user] aims \the [fired_from] at [execution_target]'s head!"))
 
-		if(!do_after(user, 1 SECONDS, INTERRUPT_ALL, BUSY_ICON_HOSTILE) || !user.Adjacent(execution_target))
-			fired_from.delete_bullet(P, TRUE)
-			return TRUE
+	user.next_move += 1.1 SECONDS //PB has no click delay; readding it here to prevent people accidentally queuing up multiple executions.
 
-		execution_target.apply_damage(damage * 3, BRUTE, "head", no_limb_loss = TRUE, permanent_kill = TRUE) //Apply gobs of damage and make sure they can't be revived later...
-		execution_target.apply_damage(200, OXY) //...fill out the rest of their health bar with oxyloss...
-		execution_target.death(create_cause_data("execution", user)) //...make certain they're properly dead...
+	if(!do_after(user, 1 SECONDS, INTERRUPT_ALL, BUSY_ICON_HOSTILE) || !user.Adjacent(execution_target))
+		fired_from.delete_bullet(P, TRUE)
+		return TRUE
 
-		execution_target.update_headshot_overlay(headshot_state) //...and add a gory headshot overlay.
+	execution_target.apply_damage(damage * 3, BRUTE, "head", no_limb_loss = TRUE, permanent_kill = TRUE) //Apply gobs of damage and make sure they can't be revived later...
+	execution_target.apply_damage(200, OXY) //...fill out the rest of their health bar with oxyloss...
+	execution_target.death(create_cause_data("execution", user)) //...make certain they're properly dead...
+	shake_camera(execution_target, 3, 4)
+	execution_target.update_headshot_overlay(headshot_state) //...and add a gory headshot overlay.
 
-		execution_target.visible_message(SPAN_HIGHDANGER(uppertext("[L] WAS EXECUTED!")), \
-			SPAN_HIGHDANGER("You WERE EXECUTED!"))
+	execution_target.visible_message(SPAN_HIGHDANGER(uppertext("[L] WAS EXECUTED!")), \
+		SPAN_HIGHDANGER("You WERE EXECUTED!"))
 
-		user.count_niche_stat(STATISTICS_NICHE_EXECUTION, 1, P.weapon_cause_data?.cause_name)
+	user.count_niche_stat(STATISTICS_NICHE_EXECUTION, 1, P.weapon_cause_data?.cause_name)
 
-		var/area/execution_area = get_area(execution_target)
+	var/area/execution_area = get_area(execution_target)
 
-		msg_admin_attack(FONT_SIZE_HUGE("[key_name(usr)] has battlefield executed [key_name(execution_target)] in [get_area(usr)] ([usr.loc.x],[usr.loc.y],[usr.loc.z])."), usr.loc.x, usr.loc.y, usr.loc.z)
-		log_attack("[key_name(usr)] battlefield executed [key_name(execution_target)] at [execution_area.name].")
+	msg_admin_attack(FONT_SIZE_HUGE("[key_name(usr)] has battlefield executed [key_name(execution_target)] in [get_area(usr)] ([usr.loc.x],[usr.loc.y],[usr.loc.z])."), usr.loc.x, usr.loc.y, usr.loc.z)
+	log_attack("[key_name(usr)] battlefield executed [key_name(execution_target)] at [execution_area.name].")
 
-		if(flags_ammo_behavior & AMMO_EXPLOSIVE)
-			execution_target.gib()
-		return TRUE
+	if(flags_ammo_behavior & AMMO_EXPLOSIVE)
+		execution_target.gib()
 	return ..()
```

Comment:
does this final return ..() make the gun fire normally after execution?

Reply:
yes